src/mgmt_conn.c

```diff
@@ -1278,6 +1278,26 @@ uzfs_zinfo_rebuild_from_clone(zvol_info_t *zinfo)
 	return (uzfs_zinfo_rebuild_start_threads(&mack, zinfo, 1));
 }
 
+/*
+ * This function returns TRUE if success can be sent to target
+ */
+static boolean_t
+start_rebuild_opcode_on_healthy_replica(zvol_info_t *zinfo, char *volname)
```

Comment:
function name is confusing. let's change it to `is_start_rebuild_opcode_on_healthy....`.